README.md

```diff
@@ -50,28 +47,39 @@ all:
 
 **Create a network**
 
-* `casper-node` & `casper-node-launcher` binaries (release) must be built before running `create-kube-network` 
+* `casper-node` & `casper-node-launcher` binaries (release) must be built before running `create-kube-network`
+
+```bash
+pushd casper-node-launcher || >&2 echo "casper-node-launcher dir expected"
+cargo build --release
+popd
+
+pushd casper-node || >&2 echo "casper-node dir expected"
+cargo build --release --package casper-node
+popd
+```
+
 * `node_count option` must match the number of nodes defined in kube-hosts.yaml
 * `genesis_in_seconds option` must be longer than the time taken to spin up the network to a running state. (A higher node count will require a higher delay until genesis)
 
-
 ```
 # 5 node network
 cp kube_hosts_examples/kube-hosts-5.yaml ./kube-hosts.yaml
-./create-kube-network --node_count 5 \
+./create-kube-network --network_name_prefix "$(aws iam get-user | jq -r .User.UserName)" \
```

Comment:
No real need here to specify aws username either... anything can go here. Change. TODO